releases/v19.1.0.md

```diff
@@ -0,0 +1,129 @@
+[**Logical Plans in the Admin UI**](../v19.1/admin-ui-statements-page.html#logical-plan) | The **Statement Details** page in the Admin UI now shows the ordered steps CockroachDB will take to execute a query (i.e., the [`EXPLAIN`](../v19.1/explain.html) output). This helps you identify bottlenecks caused by how queries are planned by our heuristic and cost-based optimizers.
+[**Cascading Replication Zones**](../v19.1/configure-replication-zones.html) | Newly created replication zones will now inherit empty values from their parent. For example, if the replication zone for a table is not explicitly set with `num_replicas`, it will inherit that value from its direct parent, whether that's the `.default` replication zone from the entire cluster or the replication zone for the database containing the table.
+[**Custom Savepoints**](../v19.1/savepoint.html#customizing-the-savepoint-name) | CockroachDB now supports custom naming of `SAVEPOINT`s for compatibility with ORMs and other third-party tools.
+
+### Backward-incompatible changes
+
+Before [upgrading to CockroachDB v19.1.0](../v19.1/upgrade-cockroach-version.html), be sure to review the following backward-incompatible changes and adjust your application as necessary.
+
+- CockroachDB no longer supports the `B'abcde'` notation to express [byte array literals](../v19.1/sql-constants.html#byte-array-literals). This notation now expresses **bit** array literals like in PostgreSQL. The `b'...'` notation remains for byte array literals.
+
+- The normalized results of certain timestamp + duration operations involving year or month durations have been adjusted to agree with the values returned by PostgreSQL.
+
+- The `CHANGEFEED` [`experimental-avro` option](../v19.1/create-changefeed.html#options) has been renamed `experimental_avro`.
+
+- Timezone abbreviations, such as `EST`, are no longer allowed when parsing or converting to a date/time type. Previously, an abbreviation would be accepted if it were an alias for the session's timezone.
+
+- The way composite [foreign key](../v19.1/foreign-key.html) matches are evaluated has changed to match the Postgres behavior. If your schema currently uses composite keys, it may require updates, since this change may affect your foreign key constraints and cascading behavior. For more details and guidance, see [this note](v2.2.0-alpha.20190114.html#composite-foreign-key-matching).
+
+- Mutation statements like [`UPDATE`](../v19.1/update.html) and [`INSERT`](../v19.1/insert.html) no longer attempt to guarantee mutation or output ordering when an `ORDER BY` clause is present. It is now an error to use `ORDER BY` without `LIMIT` with the `UPDATE` statement.
```

Comment:
This is also true of `DELETE`, docs in progress via https://github.com/cockroachdb/docs/issues/4378

Reply:
Added.